app/views/recipes/show.html.haml

```diff
@@ -0,0 +1,24 @@
+.page-header
+  %h1.text-uppercase.text-center Recipes Details
+
+.row
+  .col-md-4.col-md-offset-4
+    %p
+      %strong Name: 
+      #{@recipe.name}  
+    %p
+      %strong Preparation:
+      %br
+      #{@recipe.preparation}
+    %p
+      %strong Created On: 
+      #{@recipe.created_at.strftime('%d %B %Y')}
+    %p
+      %strong Updated On: 
+      #{@recipe.updated_at.strftime('%d %B %Y')}
+
+    .text-center 
+      %br   
+      = link_to 'Edit', edit_recipe_path(@recipe), class: 'btn btn-primary'
+      = link_to 'Back', recipes_path, class: 'btn btn-default'
+
```

Comment:
%strong Name: 
 #{@recipe.name} 
i have placed #{@recipe.name}  second line because i don't want it to be bold